src/pyOpenMS/setup.py

```
@@ -1,295 +1,48 @@
-# input-encoding: latin-1
-from __future__ import print_function
+#!/usr/bin/env python
```

Ruff (0.14.6)

Shebang is present but file is not executable

(EXE001)